<?xml version="1.0"?>
<launch>
    <!--include other launch files-->
    <include file="$(find ias0060_scitos_auclair_bryan_schneider)/launch/scitos.launch">
        <arg name="gui" value="true"/> <!--change here-->
        <arg name="headless" value="false"/>
    </include>

    <!-- Load yaml file containing map metadata to ros parameter server-->
    <rosparam command="load"
        file="$(find ias0060_scitos_auclair_bryan_schneider)/data/config/map.yaml"
        ns="/map"/>

    <!-- Load yaml file containing sensor model to ros parameter server-->
    <rosparam command="load"
        file="$(find ias0060_scitos_auclair_bryan_schneider)/data/config/sensor_model.yaml"
        ns="/sensor_model"/>

    <!--launch mapping node-->
    <node name="MappingNode" pkg="ias0060_scitos_auclair_bryan_schneider" type="OGMapping.py"
	output="screen" respawn="false"/>

    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find ias0060_scitos_auclair_bryan_schneider)/data/config/rviz/scitos_mapping.rviz" /> -->

</launch>
